scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -752,12 +754,38 @@ cleanup_stobs_dir() {
     [[ $REMOTE_STORAGE_STATUS -eq 0 ]] || run_cmd_on_local_node "umount $FAILOVER_MD_DIR" > /dev/null
 }
 
+#The following command gives us the file count on the particular node.
+#It takes 3 parameters as input
+#1. filetype : The type of file whose quantity we want to count. Example "m0trace"
+#2. directory : The location where the files of filetype are stored. Example "/var/motr/datarecovery"
+#3. start_time : The time in seconds since epoch when the script has started its execution.
+get_file_count() {
+    local file_type=$1
+    local directory=$2
+    local start_time=$3
+    filecount="$(cd $directory; find . -type f -exec stat  -c "%n %Y" {} \;| sort -n | grep $file_type | awk '{if($2>"'$start_time'") print $2; }' | wc -l)"
+    echo "$filecount"
+}
+
+#The following command is used to remove the latest file generated in the given directory
+#It takes 2 parameters as input
+#1. filetype : The type of file which is to be removed. Example "m0trace"
+#2. directory : The location where the files of filetype are stored. Example "/var/motr/datarecovery"
+remove_last_file_generated() {
+    local file_type=$1
+    local directory=$2
+    filename="$(cd $directory; ls -ltr | grep $file_type | awk '{print $9}' | tail -n 1)"
+    echo "$filename"
+    rm -f "$directory/$filename"
+}
+
 # The return statements between { .. }& are to indicate the exit status of
 # child/background process that is spawned not for the function exit status.
 # This function will run beck tool on both nodes
 run_becktool() {
     local exec_status=0
-
+    max_core_file_count=2
+    max_trace_file_count=3
```

Comment:
@ydb242
The max_trace_file_count=3 **should be increased by 1** as when LOCAL_SEG_GEN_ID and REMOTE_SEG_GEN_ID is generated using m0beck it will create one more trace file as we execute m0beck.

Currently the m0trace for the same is being created in scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh.

In future we will need to have this change as currently in other PR #147 , I'm including change where m0trace file will be created in /var/motr/datarecovery folder while getting genid.